libs/designsystem/src/lib/helpers/color-helper.ts

```diff
@@ -145,3 +145,4 @@ export interface KirbyColor extends Color {
 }
 
 export type NotificationColor = keyof typeof styles.notificationColors;
+export type MainColor = keyof typeof styles.mainColors;
```

Comment:
`$main-colors` is tagged with a deprecation in `_colors.scss` so consider not using this...